libass/ass_utils.h

```diff
@@ -52,6 +52,26 @@ int has_avx(void);
 int has_avx2(void);
 #endif
 
+typedef struct {
+    const char *str;
+    size_t len;
+} ASS_StringView;
```

Comment:
Several places in the code rely on StringView's `str` to be zero terminated. From the struct alone it may seem like StringView is a fat char pointer — without the need for zero termination. I'd prefer to have a short comment here documenting what StringView is intended to be, iiuc something like this would work:
```C
/* References a substring of some valid C string (0-terminated) */
```